### DISCUSSION TITLE
Does an update that applies the same values trigger change tracking and sync?

### DISCUSSION THREAD

**Discussion by VagueGit** [Mimetis/Dotmim.Sync#384] [Q&A] [answered]
On the client we have DMS and a SQLite database with table Foo and column int Bar = 5

On the client we run this update: UPDATE Foo SET Bar = 5

No data has changed. Does the update trigger a DMS sync?

**Comment by Mimetis** [comment, marked as answer]
The Update trigger compare all the values, if they are equals, nothing happens
